lib/supavisor/health.ex

```diff
@@ -0,0 +1,91 @@
+defmodule Supavisor.Health do
+  @moduledoc """
+  Health checking functionality
+
+  Current checks:
+  - Acceptable ERPC latencies: fails if a node has high latency to all
+  other nodes through :erpc.
+  - Database reachable: fails if can't run a simple query in the database.
+  """
+
+  require Logger
+
+  @checks [
+    acceptable_erpc_latencies: {__MODULE__, :acceptable_erpc_latencies?, []},
+    database_reachable: {__MODULE__, :database_reachable?, []}
+  ]
+
+  @task_supervisor __MODULE__.TaskSupervisor
```

Comment:
Missing this in the application.ex I believe?

Reply:
I'm starting `Supavisor.Health` there, and the child_spec for `Supavisor.Health` specifies it (it's at the bottom of the module).

Reply:
Ah ya nice!